common/djangoapps/student/views.py

```diff
@@ -314,6 +314,8 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
         CertificateStatuses.notpassing: 'notpassing',
         CertificateStatuses.restricted: 'restricted',
         CertificateStatuses.auditing: 'auditing',
```

Comment:
Yes, I believe this can be removed.  If the RC timing doesn't work we can also remove this as a fast follow